Block/WidgetInitializer.php

```diff
@@ -219,6 +254,51 @@ private function getWidgetSettings(): ?WidgetSettings
         return $widgetService->getWidgetSettings();
     }
 
+    /**
+     * @param Product $product
+     * @return float|int
+     *
+     * @throws NoSuchEntityException
+     */
+    private function getProductPrice(Product $product)
+    {
+        $price = ($product->getTypeId() === 'bundle' ?
+            $product->getPriceInfo()->getPrice('regular_price')->getMinimalPrice()->getValue() :
+            $product->getFinalPrice());
+
+        if ($product->getTypeId() === 'bundle' || !$this->isTaxEnabled()) {
```

Comment:
Bundled products will have taxes too, no?
Shouldn't we always take the price with taxes included? Even when taxes are not enabled it will be the same prices as the price without taxes

Reply:
For bundle products, regular price is always equal to the price displayed in catalog pages - if tax is enabled, regular price will contain the tax, if tax is disabled, regular price will be without tax.

Reply:
Hey @m1k3lm Let's keep the communication about the issue in the ticket. Our team will respond in the ticket itself -> https://sequra.atlassian.net/browse/LIS-13